project/CopyrightHeader.scala

```diff
@@ -61,6 +61,23 @@ trait CopyrightHeader extends AutoPlugin {
       |This file is part of the Apache Pekko project, which was derived from Akka.
       |""".stripMargin
 
+  def apacheHeader: String =
+    """Licensed to the Apache Software Foundation (ASF) under one or more
+      |contributor license agreements. See the NOTICE file distributed with
+      |this work for additional information regarding copyright ownership.
+      |The ASF licenses this file to You under the Apache License, Version 2.0
+      |(the "License"); you may not use this file except in compliance with
+      |the License. You may obtain a copy of the License at
+      |
+      |   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
not https?

Reply:
See https://github.com/apache/incubator-pekko/pull/474#discussion_r1255349026